CONTRIBUTING.md

```diff
@@ -34,6 +34,8 @@ To run only the in-crate unit tests, which don't call out to AWS, include the `-
 
 For more verbose test output, you can run `cargo test --verbose --features FEATURE -- --nocapture`.
 
+**Warning**: When building or testing with `--features all` the build/test can require upwards of 5 GB of memory. You can limit the number of features to build at once to prevent running out of memory. That can be achieved using the `--features` flag, e.g. `cargo build --features "packagea packageb"`. See [README](README.md) for a table of available services and their Cargo feature names. 
```

Comment:
Thanks for adding this to the file.  I'd like to clarify how to build certain features at once, as the example here of `cargo build --features "packagea packageb"` is currently the same as `cargo build --features all`.  One way we could keep this clear is `cargo build --features packagea && cargo build --features packageb`.